Resources/Computing/ComputingElement.py

```diff
@@ -428,7 +428,8 @@ def getDescription( self ):
           tagList = []
           for i in range( 2, cores+1 ):
             tagList.append( '%dCore' % i )
-          ceDict['Tag'] = tagList  
+          ceDict.setdefault( 'Tag', [] )
```

Comment:
I think we can announce a competition "Do your entire python program in just one line". There will be a lot of participants :)